src/NServiceBus.Transport.Msmq.TransportTests/NServiceBus.Transport.Msmq.TransportTests.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
-    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="7.1.0" />
+    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="7.2.0-beta0079"/>
```

Comment:
nitpick: Missing space between quote and the closing tag

Reply:
Good catch, fixed!